src/main/java/org/kohsuke/github/extras/okhttp3/OkHttpConnector.java

```diff
@@ -21,12 +32,11 @@
  * @author Liam Newman
  * @author Kohsuke Kawaguchi
  */
-public class OkHttpConnector implements HttpConnector {
+public class OkHttpConnector implements GitHubConnector {
```

Comment:
Darn.  `OkHttpConnector` need to continue to use the `HttpConnector` interface because it is may be extended by clients which will depend on further overriding `HttpConnector.connet()`.  

I'll need to create an `OkHttpGitHubConnector` class that contains this one.  I'll also need to retain `ObsoleteUrlFactory` for now as well.  😢